cosette/core.py

```diff
@@ -234,10 +235,11 @@ def _mk_content(src):
 # %% ../00_core.ipynb 89
 def mk_msg(content, # A string, list, or dict containing the contents of the message
            role='user', # Must be 'user' or 'assistant'
+           multimodal=True, 
            **kwargs):
     "Helper to create a `dict` appropriate for a message. `kwargs` are added as key/value pairs to the message"
     if hasattr(content, 'content'): content,role = content.content,content.role
     if isinstance(content, ChatCompletion): return find_block(content)
-    if content is not None and not isinstance(content, list): content=[content]
-    content = [_mk_content(o) for o in content] if content else ''
+    if content and not multimodal: content = " .".join(content)
```

Comment:
Is this a reasonable way to join the list? I guess "\n" would be another option.